### DISCUSSION TITLE
How optimized is the Video Player component?

### DISCUSSION THREAD

**Discussion by DrDallas1** [SmartlyDressedGames/Unturned-3.x-Community#4106] [Q&A]
Hia,

I would like to use the video player component in a pretty smart way for plenty of different gameplay components, but I wonder how optimized it is. I plan on using 20-60 video players, and I think it could be absolutely destructive to FPS. What do you guys think?

**Comment by GazziFX** [comment]
Probably not supporting hw decoding but shouldn't be a problem with medium resolutions, just test it in the game with 2-4 cores enabled

Edit: HW decoding is supported according to this [page](https://docs.unity3d.com/2021.3/Documentation/Manual/VideoSources-VideoFiles.html)

**Reply by DrDallas1** [reply]
Do you think it’s demanding client or server side? I’m not sure whether just clients or the server is impacted for the process

**Reply by GazziFX** [reply]
Only client, if you don't add it in `Clip` Prefab
